pkg/test/framework/resource/settings.go

```diff
@@ -195,6 +197,13 @@ func (s *Settings) SkipVMs() {
 	s.SkipWorkloadClasses = append(s.SkipWorkloadClasses, "vm")
 }
 
+// Checks if dual-stack mode is to be enabled for tests based on the IpFamilies
+func (s *Settings) SetDualStack() {
+	if len(strings.Split(s.IPFamilies, ",")) > 1 {
+		s.EnableDualStack = true
```

Comment:
```suggestion
		s.EnableDualStack = true
```

This test flag should go away entirely. Anything that did check it previously should just do `len(IPFamilies) > 1` instead.

Reply:
Isn't it better to parse the IP families just once and set the book dualstack flag? And everywhere it is currently being used can remain as is? Or you suggest we change this? It is a string split, and hence I chose this. I was not so clear on how to pass it as a list from the settingscommandline

Reply:
> Isn't it better to parse the IP families just once and set the book dualstack flag? 

IMO I don't think so, it just creates indirection, we already have too many settings in the test suite, we don't need to alias an existing one and create more.